src/redux/slices/videogamesSlice.js

```diff
@@ -13,17 +14,34 @@ const getDetails = createAsyncThunk('videogames/getDetails', async (id) => {
   }
 });
 
+export const getVideogames = createAsyncThunk(
+  'videogames/getVideogames',
+  async (_, { getState, rejectWithValue }) => {
+    const state = getState();
+    const url = `${BASE_URL}/videogames`;
+    const headers = { Accept: 'application/json', Authorization: state.user.jwt };
+    const { status, data, message } = await axios.get(url, { headers }).catch((error) => error);
+
+    if (status === 200) {
+      return data;
+    }
+
+    return rejectWithValue(message);
+  },
+);
+
 const videogamesSlice = createSlice({
   name: 'videogames',
   initialState: {
-    all: [],
+    videogames: [],
```

Comment:
I think I would prefer 'all' instead of 'videogames', otherwise, when we access to them into a component we will need to write something like: (state => state.videogames.videogames)